google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -99,11 +116,6 @@ public OSProcessHandler createProcessHandler(
             executor.run();
 
             Process devappserverProcess = executor.getProcess();
-            if (devappserverProcess == null) {
```

Comment:
The way it blows with a thrown unchecked exception or an ExecutionException is really similar AFAIK. The only difference is that you'll get a nasty visible stack trace, whereas with an ExecutionException you get a user friendly notification, with a red baloon, the report to google cloud dialog, etc.

This was also a requested fix from #932